cicecore/cicedynB/general/ice_forcing.F90

```diff
@@ -4394,7 +4156,7 @@ subroutine ocn_data_ncar(dt)
         do j = 1, ny_block 
           do i = 1, nx_block 
             if (n == 2) sss    (i,j,:) = c0
-            if (n == 3) hmix   (i,j,:) = c0
+            if (n == 3) hmix   (i,j,:) = mixed_layer_depth_default
```

Comment:
Maybe you're right.  I'm just implementing a more robust solution to avoid divide by zero.  I could undo the setting here, but would want to retest everything.  Let me know if you want me to do that.

Reply:
We seem to be in a transition between initializing everything in sight (what I've always done), and letting compiler flags use uninitialized values to find problems.  In my opinion, if this line is initialized to anything, it ought to be zero.  That would similarly catch places in the code that are accessing land values (and shouldn't be).  However if there's still weirdness with the initialization on padded cells or halos (isn't that how this all got started?), then leave it like this.  It looks out of place to me but I can countenance the need for it.

Reply:
Let me change this back to zero and I'll retest.  If that works, I'll push and merge.  If not, I'll just leave it as is and merge.

Reply:
Setting this to c0 is fine and I have committed that and will merge as soon as the github actions testing passes.  If there are any other questions, feel free to raise them.  thanks.